### PR TITLE
TCL update

### PR DESCRIPTION
**Pull request by jterry64**

### CODE REVIEW

File: src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalDF.scala [hunk 1]
```diff
@@ -201,14 +265,40 @@ object AnnualUpdateMinimalDF {
       max($"is__gfw_wood_fiber") as "is__gfw_wood_fiber",
       max($"is__gfw_resource_rights") as "is__gfw_resource_rights",
       max($"is__gfw_managed_forests") as "is__gfw_managed_forests",
-      max($"is__umd_tree_cover_gain") as "is__umd_tree_cover_gain"
+      max($"is__umd_tree_cover_gain") as "is__umd_tree_cover_gain",
+      max(length($"forest_age__category"))
```
[the remark this answers is not in the document]
Comment (jterry64) [reply src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalDF.scala:269]: this is just for a special "whitelist" table, which just containers true/false values if that contextual layer intersects the area of interest at all so the frontend knows whether to show it as an option in the list of filters. E.g. some layers might not cover the whole globe, so we don't want to display them for areas outside that extent.

File: src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalDF.scala [hunk 2]
```diff
@@ -166,9 +231,10 @@ object AnnualUpdateMinimalDF {
         sum("umd_tree_cover_loss__ha") as "umd_tree_cover_loss__ha",
         sum("whrc_aboveground_biomass_loss__Mg") as "whrc_aboveground_biomass_loss__Mg",
         sum("whrc_aboveground_co2_emissions__Mg") as "whrc_aboveground_co2_emissions__Mg",
-        sum("gfw_forest_carbon_gross_emissions_co2_only__Mg_CO2e") as "gfw_forest_carbon_gross_emissions_co2_only__Mg_CO2e",
-        sum("gfw_forest_carbon_gross_emissions_non_co2__Mg_CO2e") as "gfw_forest_carbon_gross_emissions_non_co2__Mg_CO2e",
-        sum("gfw_forest_carbon_gross_emissions__Mg_CO2e") as "gfw_forest_carbon_gross_emissions__Mg_CO2e",
+        sum("gfw_full_extent_gross_emissions_CO2_only__Mg_CO2") as "gfw_full_extent_gross_emissions_CO2_only__Mg_CO2",
+        sum("gfw_full_extent_gross_emissions_non_CO2__Mg_CO2e") as "gfw_full_extent_gross_emissions_non_CO2__Mg_CO2e",
+        sum("gfw_full_extent_gross_emissions__Mg_CO2e") as "gfw_full_extent_gross_emissions__Mg_CO2e",
+        sum("gfw_gross_emissions_co2e_all_gases__Mg") as "gfw_gross_emissions_co2e_all_gases__Mg"
```
[the remark this answers is not in the document]
Comment (dagibbs22) [reply src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalDF.scala:237]: Ah, right. I remember you mentioning that before.

File: src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalDF.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (dagibbs22) [reply src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalDF.scala:269]: Great. I was hoping it was like that.

File: src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalDF.scala [hunk 3]
```diff
@@ -69,19 +97,41 @@ object AnnualUpdateMinimalDF {
...
-        $"data.totalGrossEmissionsCo2eCo2Only" as "gfw_forest_carbon_gross_emissions_co2_only__Mg_CO2e",
-        $"data.totalGrossEmissionsCo2eNonCo2" as "gfw_forest_carbon_gross_emissions_non_co2__Mg_CO2e",
-        $"data.totalGrossEmissionsCo2e" as "gfw_forest_carbon_gross_emissions__Mg_CO2e"
+        $"data.totalGrossCumulAbovegroundRemovalsCo2" as "gfw_full_extent_aboveground_gross_removals__Mg_CO2",
+        $"data.totalGrossCumulBelowgroundRemovalsCo2" as "gfw_full_extent_belowground_gross_removals__Mg_CO2",
+        $"data.totalNetFluxCo2" as "gfw_full_extent_net_flux__Mg_CO2e",
+        $"data.totalGrossEmissionsCo2eCo2Only" as "gfw_full_extent_gross_emissions_CO2_only__Mg_CO2",
+        $"data.totalGrossEmissionsCo2eNonCo2" as "gfw_full_extent_gross_emissions_non_CO2__Mg_CO2e",
+        $"data.totalGrossEmissionsCo2e" as "gfw_full_extent_gross_emissions__Mg_CO2e",
+        $"data.totalSoilCarbon" as "gfw_soil_carbon_stocks__Mg_CO2",
```
[the remark this answers is not in the document]
Comment (dagibbs22) [reply src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalDF.scala]: Thanks for including `2000` in the field name and changing to C.

File: src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalDF.scala [hunk 4]
```diff
@@ -111,13 +162,19 @@ object AnnualUpdateMinimalDF {
...
-        sum("gfw_forest_carbon_gross_emissions_non_co2__Mg_CO2e") as "gfw_forest_carbon_gross_emissions_non_co2__Mg_CO2e",
-        sum("gfw_forest_carbon_gross_emissions__Mg_CO2e") as "gfw_forest_carbon_gross_emissions__Mg_CO2e",
+        sum("gfw_full_extent_aboveground_gross_removals__Mg_CO2") as "gfw_full_extent_aboveground_gross_removals__Mg_CO2",
+        sum("gfw_full_extent_belowground_gross_removals__Mg_CO2") as "gfw_full_extent_belowground_gross_removals__Mg_CO2",
+        sum("gfw_full_extent_gross_removals__Mg_CO2") as "gfw_full_extent_gross_removals__Mg_CO2",
+        sum("gfw_full_extent_net_flux__Mg_CO2e") as "gfw_full_extent_net_flux__Mg_CO2e",
+        sum("gfw_full_extent_gross_emissions_CO2_only__Mg_CO2") as "gfw_full_extent_gross_emissions_CO2_only__Mg_CO2",
+        sum("gfw_full_extent_gross_emissions_non_CO2__Mg_CO2e") as "gfw_full_extent_gross_emissions_non_CO2__Mg_CO2e",
+        sum("gfw_full_extent_gross_emissions__Mg_CO2e") as "gfw_full_extent_gross_emissions__Mg_CO2e",
+        sum("gfw_soil_carbon_stocks__Mg_CO2") as "gfw_soil_carbon_stocks__Mg_CO2",
```
[the remark this answers is not in the document]
Comment (dagibbs22) [reply src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalDF.scala]: Thanks for including `2000` in the field name and changing to C.

File: src/main/resources/raster-catalog-default.json [hunk 5]
```diff
@@ -295,6 +299,54 @@
...
       "name":"mapbox_river_basins",
       "source_uri": "s3://gfw-data-lake/mapbox_river_basins/v2018/raster/epsg-4326/{grid_size}/{row_count}/id/geotiff/{tile_id}.tif"
+    },
+    {
+      "name":"gfw_soil_carbon_stocks",
+      "source_uri": "s3://gfw-data-lake/gfw_soil_carbon_stocks/v20200724/raster/epsg-4326/{grid_size}/{row_count}/Mg_CO2e_ha/geotiff/{tile_id}.tif"
+    },
+    {
+      "name":"gfw_forest_age",
+      "source_uri": "s3://gfw-data-lake/gfw_forest_age/v20210621/raster/epsg-4326/{grid_size}/{row_count}/category/geotiff/{tile_id}.tif"
```
[the remark this answers is not in the document]
Comment (dagibbs22) [reply src/main/resources/raster-catalog-default.json]: Looks good. Thanks.

**Pull request merged**